sync/wikiman.py

```diff
@@ -47,19 +47,23 @@ def make_artifact_entry(details: Dict[str, str]) -> Dict[str, Any]:
     else:
         raise ValueError(f"'repoName' or 'url' key not specified for '{name}' in '{SOURCE_YAMLFILE}'")
 
-    return {
+    entry = {
         'name': name,
         'artifactUrl': artifact_url,
         'artifactLevel': 1,
         'destination': details['destination'],
+        'remove' : remove_from_all + details.get('remove', []),
     }
+    return entry
 
 default_branch = get_mediawiki_branch_from_version(mediawiki_version)
+remove_from_all = codebases.get('removeFromAll', [])
 output: List[Dict] = [make_artifact_entry({
     'name': 'mediawiki',
     'repoName': 'wikimedia/mediawiki',
     'repoRef': codebases.get('mediawikiRepoRef', default_branch),
-    'destination': './dist'
+    'destination': './dist',
+    'remove': codebases.get('mediawikiRemove', [])
```

Comment:
In conjunction with the suggestion above: 
```suggestion
    'remove': remove_from_all + codebases.get('mediawikiRemove', [])
```

Reply:
We would then also need to add remove_from_all to all over codebase entries before it is passed to `make_artifact_entry`
As this line is only for mediawiki

Reply:
Done differently in https://github.com/wbstack/mediawiki/pull/185/commits/2e886a9deb0af2830e28c3a84a6cd163937b21e3